src/components/AppSessionStateContext.tsx

```diff
@@ -66,12 +66,17 @@ export const AppSessionStateProvider = ({
     );
 
   useEffect(() => {
-    appSessionStateMonitor.onSessionUpdate = (newValue) =>
+    appSessionStateMonitor.onSessionUpdate = () => {
+      const newValue = appSessionStateAccessor.getCurrentSessionState();
```

Comment:
Regarding build check issue, looks like you need to add `appSessionStateAccessor` into useEffect dependency array

Reply:
Oh, thanks!